tox.ini

```diff
@@ -45,41 +60,43 @@ commands =
     python setup.py --quiet install
     discover -p "*.py" -s python/tests
 
-[testenv:examples]
-skip_install = True
-passenv =
-    RASCALINE_BUILD_TYPE
-    RUSTUP_HOME
-    CARGO*
 
+[testenv:examples]
+# Run Python examples to check that they still run correctly
 deps =
     equistore @ https://github.com/lab-cosmo/equistore/archive/513889.zip
     numpy
     chemfiles
     ase
-
 commands =
     python setup.py --quiet install
     python python/tests/check_examples.py
 
-[testenv:lint]
+
+[testenv:build-python]
+# Make sure we can build sdist and a wheel for python
 deps =
-    flake8>=4
-    flake8-bugbear
-    black
-    isort
-skip_install = true
+    setuptools
+    wheel
+    twine
+
 commands =
-    flake8 {posargs:python setup.py}
-    black --check --diff .
-    isort --check-only --diff python
+    # check building sdist and wheels from a checkout
+    python setup.py sdist
+    python setup.py bdist_wheel
+    twine check dist/*.tar.gz
+    twine check dist/*.whl
+
```

Comment:
@PicoCentauri I had to remove check-manifest since it was failing to run inside a container (see https://github.com/Luthaf/rascaline/runs/6940873009?check_suite_focus=true, which had `git config --global --add safe.directory /__w/rascaline/rascaline`).

I think it is fine since we are testing that the sdist can be used to build a wheel just below